Doc/whatsnew/3.11.rst

```diff
@@ -365,6 +365,11 @@ Other Language Changes
 * Add :option:`-P` command line option and :envvar:`PYTHONSAFEPATH` environment
   variable to not prepend a potentially unsafe path to :data:`sys.path` such as
   the current directory, the script's directory or an empty string.
+
+  Add also the :option:`-p` option to prepend the potentially unsafe path to
+  :data:`sys.path`: ignore the :option:`-P` option and the
```

Comment:
At minimum I'd use more appropriate punctuation, and make clear that "ignores" is referring to the option, not on the same level as the "add":

```suggestion
  :data:`sys.path`; ignores the :option:`-P` option and the
```

However, I'd also suggest using slightly clearer and more appropriate verbiage matching that suggested previously:

```suggestion
  :data:`sys.path`; overrides the :option:`-P` option and the
```